torch/onnx/_internal/fx/passes/type_promotion.py

```diff
@@ -171,6 +171,42 @@ def preview_type_promotion(
         )
 
 
+class DivTypePromotionRule(TypePromotionRule):
+    def __init__(self):
+        super().__init__(
+            "aten",
+            "div",
+            promote_args_positions=(0, 1),
+            promote_kwargs_names=(),
+            promotion_kind=_prims_common.ELEMENTWISE_TYPE_PROMOTION_KIND.DEFAULT,
+        )
+
+    def preview_type_promotion(
+        self, args: tuple, kwargs: dict
+    ) -> TypePromotionSnapshot:
+        """Reference type promotion rule from torch._refs.div.
+
+        Rule depends on the value of the `rounding_mode` argument.
```

Comment:
That docstring would make more sense in the class docstring instead. It explains why that class specialization was needed in the first place :)